models/ed/R/met2model.ED2.R

```diff
@@ -278,59 +271,57 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
       co2A <- CO2 * 1e+06  # surface co2 concentration [ppm] converted from mole fraction [kg/kg]
     }
 
-    ## create directory if(system(paste('ls',froot),ignore.stderr=TRUE)>0)
-    ## system(paste('mkdir',froot))
-
-    ## write by year and month
-    for (y in year + 1:nyr - 1) {
```

Comment:
This is a _nested_ year loop --- this entire section is already looping over years (based on the PEcAn standard met files). Again, I don't know if this was because of something weird related to time zones. All of the changes below are just removing the outer year loop and re-indenting -- I didn't mess with any of the underlying logic here.